sequence_Helicopter/modules/dna_rna_operations.py

```diff
@@ -0,0 +1,145 @@
+# Создадим словари комплементарности
+
+dna_complenent_dict = {
+    "A": "T",
+    "a": "t",
+    "T": "A",
+    "t": "a",
+    "G": "C",
+    "g": "c",
+    "C": "G",
+    "c": "g"
+}
+
+rna_complenent_dict = {
+    "A": "U",
+    "a": "u",
+    "U": "A",
+    "u": "a",
+    "G": "C",
+    "g": "c",
+    "C": "G",
+    "c": "g"
+}
+
+
+# Функция транскрипции - меняет в T на U
+
+def transcribe(nk_str: list[str]) -> list[str]:
+    """
+    Transcribes DNA sequences into RNA.
+
+    Arguments:
+    dna_str -- a list of strings containing DNA or RNA sequences.
+
+    Returns:
+    A list of strings containing RNA sequences.
+    """
+    # Если у нас на вход подается список с одной строкой
+    # то вынесем эту строку в новую переменную
+    if len(nk_str) == 1:
+        one_string = nk_str[0]
+        # Заменим основания методом строк .replace()
+        trans_rna = one_string.replace("T", "U").replace("t", "u")
+        return trans_rna
+    else:
+        # Заменим основания в списке методом строк .replace()
+        trans_rna = [nk.replace("T", "U").replace("t", "u") for nk in nk_str]
+        return trans_rna
+
+
+# Функция, которая разворачивает последовательность
+def reverse(nk_str: list[str]) -> list[str]:
+    """
+    Returns the reversed sequences of DNA or RNA.
+
+    Arguments:
+    nk_str -- a list of strings containing DNA or RNA sequences.
+
+    Returns:
+    A list of strings containing the reversed DNA or RNA sequences.
+    """
+    if len(nk_str) == 1:
+        one_string = nk_str[0]
+        transcript_rna = one_string[::-1]
+        return transcript_rna
+    else:
+        transcript_rna = [nk[::-1] for nk in nk_str]
+        return transcript_rna
+
+
+# Функция, которая возвращает комплементарную последовательность
+def complement(nk_str: list[str]) -> list[str]:
+    """
+    Returns the complementary sequences of DNA or RNA.
+
+    Arguments:
+    nk_str -- a list of strings containing DNA or RNA sequences.
+
+    Returns:
+    A list of strings containing the complementary sequences.
+    """
+
+    if len(nk_str) == 1:
+        for nk in nk_str:
+            if "T" in nk or "t" in nk:

```

Comment:
В этом условии (и далее) можно было бы прописать что-то из серии:
```suggestion
if "t" in nk.lower():
```